index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="mylogo">
+                <h1>
+                    My Logo
+                </h1>
+            </div>
+            <div class="options"> 
+                <a href="#">
+                    <img src="img/Union.png" alt="Union">
+                </a>
+            </div>
```

Comment:
- [ ] Kindly wrap your hamburger button in a `<nav>` tag instead of a `<div>` tag. This is for semantic and accessibility reasons.

Reply:
## Wrapped 💼
- used your advice and wrapped the anchor tag
- now its linking to the home page
- thanks for noticing

Reply:
![Horizon Node](https://media.giphy.com/media/o2LivNgy8GYqQbDdYx/giphy.gif)